src/http-api.js

```diff
@@ -385,24 +391,45 @@ module.exports.MatrixHttpApi.prototype = {
         if (!queryParams) {
             queryParams = {};
         }
-        if (!queryParams.access_token) {
-            queryParams.access_token = this.opts.accessToken;
+        if (this.useAuthorizationHeader) {
+            if (isFinite(opts)) {
+                // opts used to be localTimeoutMs
```

Comment:
I don't know what else is still relying on this old behaviour, so please can you update the docs from:
```
@param {Object=} opts additional options
```
to:
```
@param {Object|Number=} opts additional options. If a number is specified, this is treated as `opts.localTimeoutMs`.
```